### PR TITLE
Consolidate the netgear_lte configuration

### PR DESCRIPTION
**Pull request by amelchio**
**Description:**

This consolidates the configuration under the `netgear_lte` component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#22105

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/netgear_lte.markdown [hunk 1]
```diff
@@ -52,63 +60,44 @@ password:
...
+      type: string
+sensor:
+  description: Configuration options for sensors.
   required: false
-  type: string
+  type: map
+  keys:
+    monitored_conditions:
+      description: Sensor types to create.
+      required: true
```
Comment (MartinHjelmare) [inline source/_components/netgear_lte.markdown]: false

Comment (MartinHjelmare) [reply source/_components/netgear_lte.markdown]: Add `default: usage`.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

File: source/_components/netgear_lte.markdown (hunk 1 above)
Comment (amelchio) [reply source/_components/netgear_lte.markdown]: Hmm ... I would like the default to be underspecified but maybe that is not something we do?

Comment (MartinHjelmare) [reply source/_components/netgear_lte.markdown]: Why? Nah, we should be accurate in the docs.

Comment (amelchio) [reply source/_components/netgear_lte.markdown]: To discourage users from depending on it. I wanted the default to just hint that sensors are available and then users could specify the ones they actually want.

I am fine with listing the default though.

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**